src/main/java/eu/dissco/backend/controller/RestResponseEntityExceptionHandler.java

```diff
@@ -18,44 +19,79 @@ public class RestResponseEntityExceptionHandler extends ResponseEntityExceptionH
 
   @ResponseStatus(HttpStatus.UNPROCESSABLE_ENTITY)
   @ExceptionHandler(JsonProcessingException.class)
-  public ResponseEntity<String> handleJsonException(JsonProcessingException ex) {
-    return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body(ex.getMessage());
+  public ResponseEntity<ExceptionResponseWrapper> handleJsonException(JsonProcessingException e) {
+    var exceptionResponse = new ExceptionResponseWrapper(
+            HttpStatus.UNPROCESSABLE_ENTITY,
+            "Json Processing Exception",
+            e.getMessage()
+    );
+    return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body(exceptionResponse);
   }
 
   @ResponseStatus(HttpStatus.CONFLICT)
   @ExceptionHandler(ConflictException.class)
-  public ResponseEntity<Void> handleException(ConflictException e) {
-    return ResponseEntity.status(HttpStatus.CONFLICT).build();
+  public ResponseEntity<ExceptionResponseWrapper> handleException(ConflictException e) {
+    var exceptionResponse =  new ExceptionResponseWrapper(
+            HttpStatus.CONFLICT,
+            "Conflict",
+            e.getMessage()
+    );
+        return ResponseEntity.status(HttpStatus.CONFLICT).body(exceptionResponse);
   }
 
   @ResponseStatus(HttpStatus.NOT_FOUND)
   @ExceptionHandler(NotFoundException.class)
-  public ResponseEntity<Void> handleException(NotFoundException e) {
-    return ResponseEntity.status(HttpStatus.NOT_FOUND).build();
+  public ResponseEntity<ExceptionResponseWrapper> handleException(NotFoundException e) {
+    var exceptionResponse =  new ExceptionResponseWrapper(
+            HttpStatus.NOT_FOUND,
+            "Resource Not Found",
+            e.getMessage()
```

Comment:
This might also be empty, maybe check if we add a message to all exception (the ones we throw ourselves)